### DISCUSSION TITLE
Tutorial for stochastic emission of dipoles using reciprocity: n DFT monitors vs 1 volumetric DFT monitor

### DISCUSSION THREAD

**Discussion by framigo** [NanoComp/meep#2265] [Q&A] [answered]
In https://github.com/NanoComp/meep/blob/master/python/examples/stochastic_emitter_reciprocity.py,
suppose I want ndipole = the number of yee grid, instead of setting ndipole DFT monitors can I just use 1 DFT monitor which spans the entire x dimension and set yee_grid=True?

**Comment by smartalecH** [comment, marked as answer]
See the comments in #2262. 

Yes you can use a single monitor. You shouldn't need to snap to the yee grid though.

**Reply by framigo** [reply]
Great! Thanks for confirming.
